src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
```

Comment:
please see checklist - Close all resources

Reply:
done, replaced try to try-with-resources